tests/x11/libreoffice/libreoffice_pyuno_bridge_no_evolution_dep.pm

```diff
@@ -38,17 +38,17 @@ sub run {
     # navigate to the python samples item
     assert_screen 'ooffice-writer-mymacros';
     send_key 'down';
-    assert_and_dclick 'ooffice-writer-libreofficemacros';
-    send_key_until_needlematch 'ooffice-python-samples', 'down';
+    assert_and_click 'ooffice-writer-libreofficemacros';
+    wait_still_screen(2);
+    type_string "python\n";
```

Comment:
can't say I like `wait_still_screen` but if you say it's faster then I trust you.

Reply:
it's faster not because of `wait_still_screen` ...

Reply:
You can see it in log, just need to look
before:
[finished libreoffice_pyuno_bridge_no_evolution_dep x11/libreoffice at 2020-03-31 02:46:09 (102 s)](https://openqa.suse.de/tests/4064684/file/autoinst-log.txt) 15sp2
after:
[finished libreoffice_pyuno_bridge_no_evolution_dep x11/libreoffice at 2020-03-30 18:40:12 (63 s)](http://10.100.12.155/tests/14858/file/autoinst-log.txt) 15sp2

Reply:
that's a lot! :+1: